dpctl/tensor/_manipulation_functions.py

```diff
@@ -288,12 +288,7 @@ def roll(X, shift, axes=None):
     return res
 
 
-def concat(arrays, axis=0):
-    """
-    concat(arrays: tuple or list of usm_ndarrays, axis: int) -> usm_ndarray
-
-    Joins a sequence of arrays along an existing axis.
-    """
+def arrays_validation(arrays):
```

Comment:
Since this is an internal function the convention is to have its name start with a single underscore. Thus `arrays_validation` -> `_arrays_validation`.